### PR TITLE
poetry: add page

### PR DESCRIPTION
**Pull request by moredatarequired**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

Note: I used the token {{project-name}} with a hyphen instead of a dash because that's how Python projects are usually named and if feels like a new user would find that more intuitive than translating the token {{project_name}} into a concrete name like actual-project-name.

### CODE REVIEW

File: pages/common/poetry.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Interactively initialize the current directory as a new Poetry project:
+
+`poetry init`
+
+- Get the latest version of all dependencies and update poetry.lock:
+
+`poetry update`
+
+- Execute {{command}} inside the project's virtualenv:
```
Comment (zdroid) [inline pages/common/poetry.md]:
```suggestion
- Execute a command inside the project's virtual environment:
```

File: pages/common/poetry.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# poetry
+
+> Manage Python packages and dependencies.
+> More information: <https://python-poetry.org/docs>.
+
+- Create a new Poetry project in the directory {{project-name}}:
+
+`poetry new {{project-name}}`
+
+- Install {{dependency}} and its subdependencies:
```
Comment (zdroid) [inline pages/common/poetry.md]:
```suggeston
- Install a dependency and its subdependencies:
```

File: pages/common/poetry.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# poetry
+
+> Manage Python packages and dependencies.
+> More information: <https://python-poetry.org/docs>.
+
+- Create a new Poetry project in the directory {{project-name}}:
```
Comment (zdroid) [inline pages/common/poetry.md]:
```suggestion
- Create a new Poetry project in the directory with a specific name:
```

**Comment by zdroid** [comment]
I've made a minor edit before merging. Thanks for the PR!

File: pages/common/poetry.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Install a dependency and its subdependencies:
+
+`poetry add {{dependency}}`
+
+- Interactively initialize the current directory as a new Poetry project:
+
+`poetry init`
+
+- Get the latest version of all dependencies and update poetry.lock:
```
Comment (agnivade) [inline pages/common/poetry.md]: poetry.lock inside backticks(``)

File: pages/common/poetry.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# poetry
+
+> Manage Python packages and dependencies.
+> More information: <https://python-poetry.org/docs>.
+
+- Create a new Poetry project in the directory with a specific name:
```
Comment (agnivade) [inline pages/common/poetry.md]: in the _current_ directory

File: pages/common/poetry.md (hunk 4 above)
Comment (zdroid) [reply pages/common/poetry.md]: Is this an official project's style? There is no mention of it in the style guide.

Comment (owenvoke) [reply pages/common/poetry.md]: I thought we had decided against it at one point. But I can't remember.

Comment (zdroid) [reply pages/common/poetry.md]: It seems there has been no final decision there. Should a new discussion be opened?

Comment (agnivade) [reply pages/common/poetry.md]: These decisions are always tricky :P But from this https://github.com/tldr-pages/tldr/pull/3114#issuecomment-506065555, it seems like putting poetry.lock inside backticks was an unambiguous decision and nobody was particularly against it.

**Pull request merged**